### DISCUSSION TITLE
[2024/05/18] 데일리 스크럼

### DISCUSSION THREAD

**Discussion by asd42270** [10aeat/10aeat_Backend#67] [데일리 스크럼]
# 논의해야할 사항!!
- "게시글 관련해서 문의 하나 드립니다. 현재 디자이너 분들이 유지보수 게시글 와이어프레임 작업을 하고 계신데, 아래 사진처럼 구현이 되다 보니 업로드 사진의 개수를 제한하는 게 어색하다는 말씀을 해주시더라고요. 현재 사진 1개당 최대 용량 5mb, 개수는 최대 5개가 가능한 상황인데, 혹시 개수 제한을 없애고 최대 용량만 25mb로 하는 방향이 가능할까요?" -> 사진을 개수를 제한하지 않고 용량을 제한하여 올리는 방식에 대해서 어떤지 의견 남겨주세요!!
---
- 댓글 수정, 삭제 가능한가요? -> 가능하다고 했습니다.
---
- 댓글의 경우, 관리자가 자신의 댓글이 아닌 다른 댓글, 소유자가 작성한 댓글의 삭제 기능도 구현이 가능한가요?? -> 가능할 것 같다고는 했는데 이것도 의견 남겨주세요!!
---
- 부트캠프 수료 후에도 유지보수 과정을 이어 나가고 싶은데, 참여 의향 있는지? -> 이것도 의견 남겨 주세요!!

**Comment by Galmaeki** [comment]
우리 댓글 작성 API가 없네요?
댓글 수정 삭제 가능해야할것같습니다
관리자가 수정삭제 가능할것같슴다
수료후에도 별일없다면 완성도 높이는 방향으로 가볼것같슴다

**Comment by ghrltjdtprbs** [comment]
1. 저는 개인적으로 사진 개수 제한이 왜 어색한지 모르겠습니다! 대부분이 용량 제한보타는 개수제한을 두지 않나요???
2. 가능
3. 이건 건물 관리자 의미의 admin이 아니라 일반적인 게시글 프로젝트 관리자 admin 기능 구현을 원하시는 것 같은데, 아무리 관리자라도 아무런 이유없이 남의 댓글을 삭제할 수 있는 권한을 부여할 수는 없습니다. 기술적으로 가능은 합니다
4. 저는 좋습니다!
